iosxr_store_box.py

```diff
@@ -37,11 +37,24 @@
 import argparse
 from argparse import RawDescriptionHelpFormatter
 import logging
+import hashlib
 
 logger = logging.getLogger(__name__)
 set_logging()
 
 
+def generate_hash(file):
+    global hash_file
+    # SHA256 the box file and store in same location
+    sha256_hash = hashlib.sha256(open(file, 'rb').read()).hexdigest()
+    logger.debug('SHA256: %s' % sha256_hash)
+    hash_file = os.path.splitext(file)[0] + '.sha256'
```

Comment:
Consider '.sha256.txt' - as my Mac at least doesn't know how to open a `.sha256` file :)